src/Sulu/Bundle/TestBundle/Testing/SuluTestCase.php

```diff
@@ -11,311 +11,14 @@
 
 namespace Sulu\Bundle\TestBundle\Testing;
 
-use Doctrine\Common\DataFixtures\Executor\ORMExecutor;
-use Doctrine\Common\DataFixtures\ProxyReferenceRepository;
-use Doctrine\Common\DataFixtures\Purger\ORMPurger;
-use Doctrine\ORM\EntityManager;
-use PHPCR\ImportUUIDBehaviorInterface;
-use PHPCR\SessionInterface;
-use PHPCR\Util\NodeHelper;
-use Sulu\Bundle\DocumentManagerBundle\Initializer\Initializer;
-use Sulu\Bundle\PageBundle\Document\HomeDocument;
-use Sulu\Component\Content\Document\WorkflowStage;
-use Symfony\Bundle\FrameworkBundle\Client;
-use Symfony\Component\Filesystem\Filesystem;
-use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
 
-abstract class SuluTestCase extends KernelTestCase
+abstract class SuluTestCase extends WebTestCase
 {
-    private static $workspaceInitialized = false;
-
-    /**
-     * @var PHPCRImporter
-     */
-    protected $importer;
-
-    /**
-     * {@inheritdoc}
-     */
-    public static function setUpBeforeClass()
-    {
-        // enables garbage collector because symfony/phpunit-bridge disables it. see:
-        // see: https://github.com/symfony/symfony/pull/13398/files#diff-81bfee6017752d99d3119f4ddb1a09edR1
-        // see: https://github.com/symfony/symfony/pull/13398 (feature list)
-        if (!gc_enabled()) {
-            gc_enable();
-        }
-    }
-
-    protected function setUp()
-    {
-        parent::setUp();
-
-        $this->importer = new PHPCRImporter(
-            $this->getPhpcrDefaultSession(),
-            $this->getPhpcrLiveSession()
-        );
-    }
-
-    /**
-     * Close the database connection after the tests finish.
-     */
-    public function tearDown()
-    {
-        parent::tearDown();
-        // close the doctrine connection
-        foreach ($this->getContainer()->get('doctrine')->getConnections() as $connection) {
-            $connection->close();
-        }
-
-        // close the jackalope connections - can be removed when
-        // https://github.com/sulu/sulu/pull/2125 is merged.
-        // this has a negligble impact on memory usage in anycase.
-        foreach ($this->getContainer()->get('doctrine_phpcr')->getConnections() as $connection) {
-            try {
-                $connection->logout();
-            } catch (\Exception $e) {
-            }
-        }
```

Comment:
I don't remember seeing similar code to this. Don't we have to do that anymore? Or have I missed it?

Reply:
the linked pull request has been merged